examples/pipelines/fl_cross_silo_basic/config.yaml

```diff
@@ -13,13 +13,22 @@ federated_learning:
     datastore: "workspaceblobstore"
 
   silos:
-    - compute: cpu-cluster-silo-1
-      datastore: workspaceblobstore
-      training_data_path: "azureml://datastores/workspaceblobstore/paths/azureml/ee95ed94-03d5-4b85-ad5e-883880b44735/extracted_data/"
-    - compute: cpu-cluster-silo-2
-      datastore: workspaceblobstore
-      training_data_path: "azureml://datastores/workspaceblobstore/paths/azureml/ee95ed94-03d5-4b85-ad5e-883880b44735/extracted_data/"
+    - compute: silo-wus2
+      datastore: workspaceblobstoresilo1
+      training_data_path: "https://azureopendatastorage.blob.core.windows.net/mnist/processed/train.csv"
+      testing_data_path: "https://azureopendatastorage.blob.core.windows.net/mnist/processed/t10k.csv"
+    - compute: silo-eus2
+      datastore: workspaceblobstoresilo2
+      training_data_path: "https://azureopendatastorage.blob.core.windows.net/mnist/processed/train.csv"
```

Comment:
as discussed over Teams, please reference the data asset explicitly, not the underlying uri. Something like:
training_data_path: azureml:mnist-train-02:4

a.k.a. azureml:<data-asset-name>:<version>